src/waterfall/waterfallplot.h

```diff
@@ -118,7 +118,6 @@ class WaterfallPlot : public Waterfall
     void minDepthToDrawChanged();
     void mouseColumnConfidenceChanged();
     void mouseColumnDepthChanged();
-    void mouseDepthChanged();
```

Comment:
You are right, but the problem is different, the function should not exist in `waterfall.h` since it's only a base class for "waterfall like" plots. Such plot can not provide depth values, like what we are having in Ping360. 
TLDR: the problem is in waterfall base class, the signal should be removed there.